pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/TopicPoliciesTest.java

```diff
@@ -2641,4 +2642,23 @@ public void testDoNotCreateSystemTopicForHeartbeatNamespace() {
         });
     }
 
+    @Test(timeOut = 30000)
```

Comment:
Please add the following unit tests:
1)The namespace and topic level policies are set at the same time, whether the priority is correct

2)Disabling topic-level policies

Reply:
done.